### PR TITLE
Desynths

### PR DESCRIPTION
**Pull request by Dirk-Dieters**
For this round, I used the following criteria:

1: Item has a specific desynth recipe and yield listed on its BGWiki page (without a level) AND Item has a synthesis recipe
2: Item was previously updated in level but not activated due to yield mismatch.

In cases where the item is craftable, but the desynth recipe had no level assigned to it, the level was set to match the synthesis level, and a comment was added in case future evidence proves that the level should be different.

In cases where the yield shown on FFXIAH/the current database were similar, but not exact matches, (More HQ result information on the BGWiki page primarily), the BGWiki data was used. This includes previously identified mismatched yields, and they have been activated as well.

Some notes: 
Apparently Desynths aren't a huge priority for the BGWiki contributors, especially smithing, we might have to accept data from FFXIclopedia or another source in order to resolve many of these.

I left comments on several still-disabled recipes that I was unsure about how to proceed.

**_I affirm:_**
- [X] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [X] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by ibm2431** [comment]
End of line comments are fine if there's a space after the `--`.
`-- This is fine`.
`--This upsets SQL.`

**Comment by Dirk-Dieters** [comment]
I see, thanks

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -622,57 +622,57 @@ INSERT INTO `synth_recipes` VALUES (5508,0,0,115,70,0,0,0,0,0,0,4098,4240,1346,4
...
--- INSERT INTO `synth_recipes` VALUES (9910,1,0,255,0,0,0,0,0,0,0,4100,4242,17097,0,0,0,0,0,0,0,707,707,707,707,1,1,2,2,'Elm Pole (desynth)');
--- INSERT INTO `synth_recipes` VALUES (9911,1,0,255,0,0,0,0,0,0,0,4100,4242,17152,0,0,0,0,0,0,0,817,712,712,712,4,1,1,1,'Shortbow (desynth)');
--- INSERT INTO `synth_recipes` VALUES (9912,1,0,255,0,0,0,0,0,0,0,4100,4242,17523,0,0,0,0,0,0,0,651,711,711,711,1,1,1,1,'Quarterstaff (desynth)');
--- INSERT INTO `synth_recipes` VALUES (9913,1,0,255,0,0,0,0,0,0,0,4100,4242,12290,0,0,0,0,0,0,0,708,650,708,708,1,1,2,2,'Maple Shield (desynth)');
+INSERT INTO `synth_recipes` VALUES (9904,1,0,54,0,0,0,0,0,0,0,4100,4242,17030,0,0,0,0,0,0,0,649,649,717,717,1,1,1,1,'Great Club (desynth)'); -- Level based on level to synth
+INSERT INTO `synth_recipes` VALUES (9905,1,0,23,0,0,0,0,0,0,0,4100,4242,17051,0,0,0,0,0,0,0,841,713,713,713,1,1,1,1,'Yew Wand (desynth)'); -- Level based on level to synth
+INSERT INTO `synth_recipes` VALUES (9906,1,0,102,0,0,0,0,0,0,0,4100,4242,17592,0,0,0,0,0,0,0,771,1228,1228,1225,1,4,4,4,'Kinkobo (desynth)'); -- Level based on level to synth
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: Let's adjust the HQ1 to Yellow Rock while we're here (Kinkobo).

File: sql/synth_recipes.sql [hunk 2]
```diff
@@ -622,57 +622,57 @@ INSERT INTO `synth_recipes` VALUES (5508,0,0,115,70,0,0,0,0,0,0,4098,4240,1346,4
...
--- INSERT INTO `synth_recipes` VALUES (9920,1,0,255,0,0,0,0,0,0,0,4100,4242,17052,0,0,0,0,0,0,0,710,847,847,847,1,1,1,1,'Chestnut Wand (desynth)');
--- INSERT INTO `synth_recipes` VALUES (9921,1,0,255,0,0,0,0,0,0,0,4100,4242,17388,0,0,0,0,0,0,0,707,820,820,820,1,1,1,1,'Fastwater F. Rod (desynth)');
+INSERT INTO `synth_recipes` VALUES (9917,1,0,70,0,0,0,0,0,0,0,4100,4242,12359,0,0,0,0,0,0,0,1234,1234,1234,1234,4,6,6,6,'Hickory Shield (desynth)'); -- Level based on level to synth
+INSERT INTO `synth_recipes` VALUES (9918,1,0,34,0,0,0,0,0,0,0,4100,4242,12292,0,0,0,0,0,0,0,717,717,651,651,1,2,1,1,'Mahogany Shield (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (9919,1,0,255,0,0,0,0,0,0,0,4100,4242,17292,0,0,0,0,0,0,0,715,718,819,819,1,1,1,1,'Long Boomerang (desynth)');
+INSERT INTO `synth_recipes` VALUES (9920,1,0,32,0,0,0,0,0,0,0,4100,4242,17052,0,0,0,0,0,0,0,710,710,847,847,1,1,1,1,'Chestnut Wand (desynth)'); -- Level based on level to synth
+INSERT INTO `synth_recipes` VALUES (9921,1,0,51,0,0,0,0,0,0,0,4100,4242,17388,0,0,0,0,0,0,0,707,707,820,820,1,1,1,1,'Fastwater F. Rod (desynth)'); -- Level based on level to synth
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: 51 is for the "restoral" to fix a broken rod; the synth recipe from the desynth ingredients  ingredients is 56~

File: sql/synth_recipes.sql [hunk 3]
```diff
@@ -2023,43 +2023,44 @@ INSERT INTO `synth_recipes` VALUES (25514,0,0,0,0,118,0,0,0,0,0,4096,4238,3323,8
 -- INSERT INTO `synth_recipes` VALUES (25527,0,2004,0,0,255,0,0,0,0,0,4096,4238,4076,9250,16408,0,0,0,0,0,21513,21514,21514,21514,1,1,1,1,'Raetic Baghnakhs');
 -- INSERT INTO `synth_recipes` VALUES (25528,0,0,0,255,255,0,0,0,0,0,4096,4238,4077,4081,9248,16605,0,0,0,0,21606,21607,21607,21607,1,1,1,1,'Enriching Sword');
 -- INSERT INTO `synth_recipes` VALUES (25529,0,2005,0,0,255,0,255,0,0,0,4102,4244,9003,9062,9251,9832,9844,9845,9845,9852,21628,21629,21630,21630,1,1,1,1,'Valor Sword');
--- INSERT INTO `synth_recipes` VALUES (29901,1,0,0,0,255,0,0,0,0,0,4100,4242,13473,0,0,0,0,0,0,0,814,744,744,744,1,2,2,2,'Amber Ring (desynth)'); --yield mismatch
+
+INSERT INTO `synth_recipes` VALUES (29901,1,0,0,0,55,0,0,0,0,0,4100,4242,13473,0,0,0,0,0,0,0,814,814,744,744,1,1,1,2,'Amber Ring (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql:2026]: BGWiki is listing an earth crystal for the desynth, but I'm fairly certain we can reasonably ignore what was probably an editor copy-pasting from the synth table, and keep this lightning crystal instead.

File: sql/synth_recipes.sql [hunk 4]
```diff
@@ -2023,43 +2023,44 @@ INSERT INTO `synth_recipes` VALUES (25514,0,0,0,0,118,0,0,0,0,0,4096,4238,3323,8
...
--- INSERT INTO `synth_recipes` VALUES (29907,1,0,0,0,55,0,0,0,0,0,4100,4242,13444,0,0,0,0,0,0,0,807,744,744,744,1,2,2,2,'Sardonyx Ring (desynth)'); --yield mismatch
--- INSERT INTO `synth_recipes` VALUES (29908,1,0,0,0,55,0,0,0,0,0,4100,4242,13483,0,0,0,0,0,0,0,792,653,653,653,1,2,2,2,'Pearl Ring (desynth)'); --yield mismatch
--- INSERT INTO `synth_recipes` VALUES (29909,1,0,0,0,61,0,0,0,0,0,4100,4242,12801,0,0,0,0,0,0,0,851,653,653,653,1,2,2,2,'Mythril Cuisses (desynth)'); --yield mismatch
--- INSERT INTO `synth_recipes` VALUES (29910,1,0,0,0,55,0,0,0,0,0,4100,4242,13479,0,0,0,0,0,0,0,811,653,653,653,1,2,2,2,'Ametrine Ring (desynth)'); --yield mismatch
+INSERT INTO `synth_recipes` VALUES (29905,1,0,0,0,55,0,0,0,0,0,4100,4242,13481,0,0,0,0,0,0,0,815,815,653,653,1,1,1,2,'Sphene Ring (desynth)');
+INSERT INTO `synth_recipes` VALUES (29906,1,0,0,0,55,0,0,0,0,0,4100,4242,13476,0,0,0,0,0,0,0,788,788,653,653,1,1,1,2,'Peridot Ring (desynth)');
+INSERT INTO `synth_recipes` VALUES (29907,1,0,0,0,55,0,0,0,0,0,4100,4242,13444,0,0,0,0,0,0,0,807,807,744,744,1,1,1,2,'Sardonyx Ring (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: BGWiki is claiming 35; Sardonyx is a STAT+1, not a STAT+2 like the Peridot and Sphene Rings before it in this cluster

File: sql/synth_recipes.sql [hunk 5]
```diff
@@ -2023,43 +2023,44 @@ INSERT INTO `synth_recipes` VALUES (25514,0,0,0,0,118,0,0,0,0,0,4096,4238,3323,8
...
+INSERT INTO `synth_recipes` VALUES (29908,1,0,0,0,55,0,0,0,0,0,4100,4242,13483,0,0,0,0,0,0,0,792,653,653,653,1,2,2,2,'Pearl Ring (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: We can correct the HQ yields while we're here:
```
Pearl,
Pearl,
Mythril Ingot
Mythril Ingot x2
```

File: sql/synth_recipes.sql [hunk 6]
```diff
@@ -3178,12 +3179,12 @@ INSERT INTO `synth_recipes` VALUES (49923,1,0,0,0,0,0,30,0,0,0,4100,4242,12826,0
 -- INSERT INTO `synth_recipes` VALUES (49930,1,0,0,0,0,0,255,0,0,0,4100,4242,14846,0,0,0,0,0,0,0,816,855,1680,1680,1,1,1,1,'Sha\'ir Gages (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49931,1,0,0,0,0,0,255,0,0,0,4100,4242,14323,0,0,0,0,0,0,0,850,819,819,819,1,5,5,5,'Noct Brais (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49932,1,0,0,0,0,0,255,0,0,0,4100,4242,13192,0,0,0,0,0,0,0,850,651,651,651,1,2,2,2,'Leather Belt (desynth)');
--- INSERT INTO `synth_recipes` VALUES (49933,1,0,0,0,0,0,255,0,0,0,4100,4242,13939,0,0,0,0,0,0,0,850,851,851,851,1,1,1,1,'Austere Hat (desynth)');
+INSERT INTO `synth_recipes` VALUES (49933,1,0,0,0,0,0,255,0,0,0,4100,4242,13939,0,0,0,0,0,0,0,850,851,851,851,1,1,1,1,'Austere Hat (desynth)'); -- BG data incomplete but does not contradict
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: BGWiki does say a synth/desynth level of 93 though 😉

File: sql/synth_recipes.sql [hunk 7]
```diff
@@ -3178,12 +3179,12 @@ INSERT INTO `synth_recipes` VALUES (49923,1,0,0,0,0,0,30,0,0,0,4100,4242,12826,0
...
 -- INSERT INTO `synth_recipes` VALUES (49934,1,0,0,0,0,0,255,0,0,0,4100,4242,12703,0,0,0,0,0,0,0,850,851,506,506,2,1,1,1,'Coeurl Gloves (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49935,1,0,0,0,0,0,255,0,0,0,4100,4242,12440,0,0,0,0,0,0,0,850,850,850,850,1,1,1,1,'Leather Bandana (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49936,1,0,0,0,0,0,255,0,0,0,4100,4242,13704,0,0,0,0,0,0,0,850,506,1117,1117,1,1,1,1,'Ogre Mask (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49937,1,0,0,0,0,0,255,0,0,0,4100,4242,13592,0,0,0,0,0,0,0,817,852,869,869,1,1,1,1,'Lizard Mantle (desynth)');
--- INSERT INTO `synth_recipes` VALUES (49938,1,0,0,0,0,0,255,0,0,0,4100,4242,13203,0,0,0,0,0,0,0,850,924,924,924,1,1,1,1,'Barbarian\'s Belt (desynth)');
+INSERT INTO `synth_recipes` VALUES (49938,1,0,0,0,0,0,255,0,0,0,4100,4242,13203,0,0,0,0,0,0,0,850,924,924,924,1,1,1,1,'Barbarian\'s Belt (desynth)'); -- BG data incomplete but does not contradict
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: Has a level of 41 though~!

File: sql/synth_recipes.sql [hunk 8]
```diff
@@ -3565,12 +3566,12 @@ INSERT INTO `synth_recipes` VALUES (55524,0,0,0,0,0,0,0,117,0,0,4099,4241,9003,9
...
--- INSERT INTO `synth_recipes` VALUES (59901,1,0,0,0,0,0,0,23,0,0,4100,4242,16649,0,0,0,0,0,0,0,715,893,893,893,1,1,1,1,'Bone Pick (desynth)'); --yield mismatch
+INSERT INTO `synth_recipes` VALUES (59901,1,0,0,0,0,0,0,23,0,0,4100,4242,16649,0,0,0,0,0,0,0,715,715,893,893,1,1,1,1,'Bone Pick (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (59902,1,0,0,0,0,0,0,255,0,0,4100,4242,17361,0,0,0,0,0,0,0,894,902,902,902,1,1,1,1,'Crumhorn (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (59903,1,0,0,0,0,0,0,255,0,0,4100,4242,13323,0,0,0,0,0,0,0,894,744,744,744,1,2,2,2,'Beetle Earring (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (59904,1,0,0,0,0,0,0,255,0,0,4100,4242,17362,0,0,0,0,0,0,0,894,883,883,883,1,1,1,1,'Shofar (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (59905,1,0,0,0,0,0,0,255,0,0,4100,4242,13457,0,0,0,0,0,0,0,894,894,894,894,1,1,1,1,'Beetle Ring (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59906,1,0,0,0,0,0,0,18,0,0,4100,4242,12454,0,0,0,0,0,0,0,880,880,880,880,1,1,1,1,'Bone Mask (desynth)'); --yield mismatch
+INSERT INTO `synth_recipes` VALUES (59906,1,0,0,0,0,0,0,18,0,0,4100,4242,12454,0,0,0,0,0,0,0,880,880,850,8,1,1,1,1,'Bone Mask (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: I suspect you don't mean to have an HQ3 of Royal Bed coming out of this Bone Mask 😆

**Review by ibm2431 — CHANGES_REQUESTED** [review]
Sorry this took me a bit to review!

Mostly okay, just a couple things I spotted.

I do appreciate someone taking the time to go through these and making the super safe changes! 🎉

**Comment by Dirk-Dieters** [comment]
Trying to figure out where to go from here, There's a lot of just baseline desynths that are still disabled, but can't be fully justified with only BGWiki and FFXIAH as sources. Do we start accepting FFXIclopedia? If so, do we take it for results, or just levels?

From what I understand, everything craftable should be desynthable at the same level as the craft, so I'll make that the goal of my next PR, but there's not justification for all those recepies on BGWiki, and we already know that FFXIAH doesn't have complete data, so I'll have to do verification from FFXIclopedia or another source.

**Review by ibm2431 — APPROVED** [review]
Kinkobo desynth's Yellow Rock HQ1 should be quantity 1, but I'll ninja-change that after merge so you don't need to wait longer on merge after a one line change. 😉

**Pull request merged**